applications/nrf5340_audio/README.rst

```diff
@@ -1294,6 +1294,9 @@ You can test upgrading the firmware on both cores at the same time on a headset
 #. Transfer the generated file to your Android or iOS device, depending on the DFU scenario.
    See the `FOTA build files`_ section for information about FOTA file name patterns.
    For transfer, you can use cloud services like Google Drive for Android or iCloud for iOS.
+   The device will reset when the transfer is complete.
+#. During startup, press **BTN 4** for a couple of seconds to turn on the DFU mode.
```

Comment:
I think this can just be something like:
"To enter DFU mode, hold down **BTN 4** while pressing **RESET**, and then continue to hold down **BTN 4** for a couple of seconds"